utils/micro.py

```diff
@@ -111,15 +112,15 @@ def get_identities_load(config):
     logging.info("Loading identities finished!")
 
 
-def get_enrich(config, backend_section):
+def get_enrich(config, backend_section, repos_to_check=None):
```

Comment:
This parameter should be added to the documentation.

Reply:
Fixed and fixed!  Thanks for the catch!